examples/docker-simple-on-ubuntu-managed-disks/terraform.tfvars

```diff
@@ -0,0 +1,11 @@
+resource_group_name = "AzureQuickstart"
+
+resource_group_location = "West US"
+
+admin_username = "azure"
+
+admin_ssh_publickey = "REPLACE_WITH_SSH_PUBLICKEY"
```

Comment:
_same as above_ - given this is an example, would it make more sense to make this a file path and use `~/.ssh/id_rsa.pub` as the default?